src/autopas/containers/linkedCells/LinkedCells.h

```diff
@@ -545,6 +557,11 @@ class LinkedCells : public CellBasedParticleContainer<FullParticleCell<Particle>
    * load estimation algorithm for balanced traversals.
    */
   autopas::LoadEstimatorOption _loadEstimator;
+
+  /**
+   * Skin distance a particle is allowed to move in one time-step.
+   */
+  double _skinPerTimestep;
```

Comment:
isn't this relevant for all containers now and thus could be moved into the `ParticleContainerInterface` (this would also affect the container constructors)?

Reply:
done in [ca2c857](https://github.com/AutoPas/AutoPas/pull/830/commits/ca2c857cd3043cf4d91f215e152900d71de0cea7)